Tetrio.Foxhole.Database/ChallengeGenerator.cs

```diff
@@ -203,6 +190,31 @@ private async Task<Challenge> GenerateChallenge(Difficulty difficulty, TetrioCon
 
                 if (value > 0)
                 {
+                    // Apply penalty based on mods and round it if it's needed
+                    if (condition is ConditionType.Apm or ConditionType.Vs)
+                        Console.WriteLine(value);
+                    if (condition is ConditionType.Pps or ConditionType.Apm or ConditionType.Vs or ConditionType.Finesse)
+                    {
+                        var mult = 1d;
+                        switch (condition)
+                        {
+                            case ConditionType.Pps:
+                                mult = .2d;
+                                break;
+                            case ConditionType.Vs:
+                                mult = .25d;
+                                break;
+                        }
+                        if (condition is not ConditionType.Finesse)
+                            value -= value * (1d - penalty) * mult;
+                        value = Math.Round(value, 2);
+                    }
+                    else
+                    {
+                        //Just round the value fully if it should be a whole number. Those also mostly don't need nerfs.
+                        //ZhunGamer did said though that challenges with gravity could use a spin count nerf.
+                        value = Math.Round(value);
+                    }
```

Comment:
I notice this in this block extremly so I only mentioned it here once as an example.

There is a big lack of new lines which makes every thing look cluttered and together even tho it isn't.

- Before and after the `switch` statement should be a new line
- Same rule goes for single `if` statements (if statements without `else` blocks)
- When starting a comment with `//` there should be a space next and then the actual comment. So instead of `//Just` it will be `// Just` and so on and so forth